packages/auth/jest.setup.js

```diff
@@ -20,6 +20,13 @@ module.exports = async (globalConfig) => {
       .withEnvironment({
         POSTGRES_PASSWORD: 'password'
       })
+      .withHealthCheck({
+        test: ['CMD-SHELL', 'pg_isready -d testing'],
+        interval: 10000,
+        timeout: 5000,
+        retries: 5
+      })
```

Comment:
same as our backend, which was added for the same reason in https://github.com/interledger/rafiki/issues/1099

https://github.com/interledger/rafiki/blob/81f8a059229d4d8e1ce714cf12605f5a17d127d3/packages/backend/jest.setup.ts#L23-L27